tests/steps/qdrant/e2e_test.py

```diff
@@ -61,25 +71,55 @@ def test_qdrant_connector_one_no_csv(input_output_folder: tuple[Path, Path]):
 
 def test_qdrant_collection_retirement(input_output_folder: tuple[Path, Path], env, dummy_collection):
     input_path, output_path = input_output_folder
-    HIST_LEN = 3
+    HIST_LEN = 10
     env.set("COLLECTION_HISTORY_LEN", str(HIST_LEN))
+
     input_file = input_path / "qdrant_at.csv"
     output_file = output_path / "QdrantConnectorStep"
     shutil.copy("./tests/data/embedded.csv", input_file)
+
     client = QdrantClient(location=":memory:")
     old_close = client.close
     client.close = print
-    with unittest.mock.patch("wurzel.steps.qdrant.step.QdrantClient") as mock:
-        mock.return_value = client
+
+    old_time = (datetime.now(timezone.utc) - timedelta(days=10)).strftime("%Y-%m-%dT%H:%M:%S.%fZ")
+    recent_time = (datetime.now(timezone.utc) - timedelta(hours=6)).strftime("%Y-%m-%dT%H:%M:%S.%fZ")
+
+    def mock_get_telemetry(*args, **kwargs):
+        # Emulate 4 collections — one will be aliased, one recently used, one old and deletable
+        return {
+            "result": {
+                "collections": {
+                    "collections": [
+                        {"id": "austria_v1",
+                         "shards": [{"local": {"optimizations": {"optimizations": {"last_responded": old_time}}}}]},
+                        {"id": "austria_v2", "shards": [{"remote": [{"searches": {"last_responded": old_time}}]}]},
+                        {"id": "austria_v3", "shards": [{"remote": [{"searches": {"last_responded": recent_time}}]}]},
+                        {"id": "austria_v4", "shards": [],
+                         },
+                    ]
+                }
+            }
+        }
+
+        with unittest.mock.patch("wurzel.steps.qdrant.step.QdrantClient") as mock:
+            mock.return_value = client
+        client.get_telemetry = mock_get_telemetry
+
+        # Simulate four executions to create 4 collections
         with BaseStepExecutor() as ex:
             ex(QdrantConnectorStep, {input_path}, output_file)
-            ex(QdrantConnectorStep, {input_path}, output_file)
-            ex(QdrantConnectorStep, {input_path}, output_file)
-            # this will cover retire
-            ex(QdrantConnectorStep, {input_path}, output_file)
+        ex(QdrantConnectorStep, {input_path}, output_file)
+        ex(QdrantConnectorStep, {input_path}, output_file)
+        ex(QdrantConnectorStep, {input_path}, output_file)
```

Comment:
they have to be in the context of with still

Reply:
and by setting the value to 10 you changed the test. its now only testing telemetry, not the traditional case, any longer